typescript/vrotsc/e2e/cases/canvas-items/workflow.wf.ts

```diff
@@ -19,6 +19,9 @@ import { Workflow, Out, In, Item, RootItem, DecisionItem, WaitingTimerItem, Work
 		},
 		result: {
 			type: "number"
+		},
```

Comment:
Why modify this test? I see you may be confused but this test is to test the `WorkflowItem`

Reply:
Actually this serves as an integration test for the currently supported components.